config/routes.rb

```diff
@@ -112,11 +112,17 @@
       get :archive, path: :arkiv, on: :collection
       get :matrix, path: :matris, on: :collection
       get 'modal/:date', action: :modal, as: :modal, on: :member
+      resources :adventures, path: :aventyr, only: [:index, :show] do
+        get 'archive', on: :collection, path: 'arkiv'
```

Comment:
Most other paths are done with symbols, like `get :archive, on: :collection, path: :arkiv`.
There is some difference for parsing strings or symbols, but it probably doesn't make any big difference here.